src/Resources/translations/SonataDashboardBundle.fr.xliff

```diff
@@ -178,26 +178,6 @@
                 <source>blocks</source>
                 <target>widgets</target>
             </trans-unit>
-            <trans-unit id="44" resname="compose.preview_error">
```

Comment:
Can you add the missing translation keys @greg0ire @Soullivaneuh ?

Reply:
Ok I'll do a PR rightaway